app/src/main/java/org/sopt/cgv/core/designsystem/component/chip/Chip.kt

```diff
@@ -0,0 +1,95 @@
+package org.sopt.cgv.core.designsystem.component.chip
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.cgv.R
+import org.sopt.cgv.core.designsystem.theme.Black
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+import org.sopt.cgv.core.designsystem.theme.Gray100
+import org.sopt.cgv.core.designsystem.theme.Gray300
+import org.sopt.cgv.core.designsystem.theme.Gray700
+import org.sopt.cgv.core.designsystem.theme.Gray850
+
+@Composable
+fun Chip(
+    modifier: Modifier = Modifier,
+    content: String,
+    inTime: Boolean = false,
+    onTheaterSelected: ((String) -> Unit)? = null
+) {
```

Comment:
```suggestion
fun Chip(
    content: String,
    modifier: Modifier = Modifier,
    inTime: Boolean = false,
    onTheaterSelected: ((String) -> Unit)? = null
) {
```
저도 이 부분을 많이 놓치는데,
Modifier 매개변수는 반드시 첫 번째 선택적 매개변수여야 합니다!!
(주의 경고를 한번 봐주시면 좋을 거 같아요)